whatwg.org/irc

```diff
@@ -58,6 +58,10 @@
 
 <p>The simplest way to get started with IRC, if you are not familiar, is by signing up for a free <a href="https://www.irccloud.com/">IRCCloud</a> account. Once you've done that, you should be logged in to the Freenode server by default. All you'll have to do is join the #whatwg channel. If you tell your browser to use IRCCloud for irc:// links, then just <a href="ircs://irc.freenode.net/whatwg">clicking this link</a> should also take you there.</p>
 
+<p>Also, it is highly recommended that you register your nick on Freenode. For more information, read <a href="https://freenode.net/kb/answer/registration">this page</a>.</p>
```

Comment:
Link "register your nick on Freenode" instead so you don't need the additional sentence. Also, this is what you _have_ to do before joining the #whatwg channel so I'd suggest merging it into the paragraph above. The channel only allows registered users these days (due to +r as channel mode).